test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointSmokeTest.java

```diff
@@ -567,7 +577,7 @@ public void MultiCheckpointWriterTestInner(boolean consecutiveCkpoints) throws E
         mcw2.addMap((SMRMap) mA);
         mcw2.addMap((SMRMap) mB);
         long firstGlobalAddress2 = mcw2.appendCheckpoints(r, author).getSequence();
-        assertThat(firstGlobalAddress2).isGreaterThan(firstGlobalAddress1);
+        assertThat(firstGlobalAddress2).isGreaterThanOrEqualTo(firstGlobalAddress1);
```

Comment:
Why isn't this a more strict check ? (i.e. just equality)

Reply:
Because this method serves the purpose of two different tests (L.571) so in the case of consecutive checkpoints it will be greater cause the stream tail will change, if not, it will be equal cause the stream tail won't change.